recipe/meta.yaml

```diff
@@ -32,18 +32,14 @@ requirements:
     - nanobind >=1.9.2
     - scikit-build-core >=0.9.3
     - numpy
-    - ocl-icd      # [linux]
-    - khronos-opencl-icd-loader      # [win or osx]
-
+    - libopencl-devel
   run:
     - python
     - pytools
-        # for pkg_resources
+    # for pkg_resources
     - setuptools
```

Comment:
```suggestion
```
(We've switched to `importlib.resources`, which should offer everything needed as of Python 3.9.)